ext/rotoscope/rotoscope.c

```diff
@@ -103,9 +103,12 @@ static rs_class_desc_t tracearg_class(rb_trace_arg_t *trace_arg) {
   VALUE klass;
   VALUE self = rb_tracearg_self(trace_arg);
 
-  if (RB_TYPE_P(self, T_MODULE) || RB_TYPE_P(self, T_OBJECT) ||
-      RB_TYPE_P(self, T_CLASS)) {
-    klass = RBASIC_CLASS(self);
+  if (RB_TYPE_P(self, T_MODULE) || RB_TYPE_P(self, T_OBJECT)) {
+    klass = CLASS_OF(self);
```

Comment:
I also switched from `RBASIC_CLASS` to `CLASS_OF` here. Only difference is `CLASS_OF` has some [short circuits](https://github.com/ruby/ruby/blob/22224a45df6eef9114799a3fa071280f77bd3d53/include/ruby/ruby.h#L1970-L1984) based on predicate checks that make it potentially a tiny bit faster than [RBASIC_CLASS](https://github.com/ruby/ruby/blob/22224a45df6eef9114799a3fa071280f77bd3d53/include/ruby/ruby.h#L878).